ivy/functional/frontends/paddle/nn/functional/common.py

```diff
@@ -85,3 +85,9 @@ def zeropad2d(x, padding, data_format="NCHW", name=None):
     else:
         raise ValueError("Unknown data_format: {}".format(data_format))
     return ivy.pad(x, padding, mode="constant", constant_values=0.0)
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+def linear(x, weight, /, *, bias=None, name=None):
```

Comment:
We don't add `/, *` in the frontend functions signatures. We just copy the signature of the original function as is.